cmd/root.go

```diff
@@ -141,3 +142,39 @@ missing parameters using the profile variables or use the appropriate flag: --%s
 		os.Exit(1)
 	}
 }
+
+func usageTemplate() string {
```

Comment:
did you come up with this on your own?

Reply:
yep, it's the Cobra's common template modified to our needs